### PR TITLE
doc: Added link to the video instructions on beginning development page

### PR DESCRIPTION
**Pull request by SteLeo1602**
Added link to the playlist of my video tutorials
image

### CODE REVIEW

**Comment by romani** [comment]
Please apply patch https://dev.azure.com/romanivanovjr/romanivanovjr/_build/results?buildId=21348&view=logs&j=6a833393-c551-5000-0916-5000fd3b86e7&t=bdbbd129-77f0-5d73-ce7d-db58d34c8358 , to let CI be green.
This is unfortunate side effect of spell checker execution over URLs, we need to add all such "words" to whitelist file.

Patch command is copy paste able to terminal.

**Review by romani — APPROVED** [review]
ok to merge is CI is green

**Comment by romani** [comment]
GitHub, generate website

**Review by nrmancuso — APPROVED** [review]
Thanks @SteLeo1602

**Comment by romani** [comment]
This is initial base set of videos, there will be more, on how to debug, on .... .

**Comment by romani** [comment]
GitHub, generate website

**Comment by nrmancuso** [comment]
> This is initial base set of videos, there will be more, on how to debug, on .... . They will be added to this play list.

@romani If this is in response to my comment at https://github.com/checkstyle/checkstyle/pull/14865#issuecomment-2100597391, you are missing the point. If you watch the video, you will notice that the actual steps for forking the repository are glossed over because it has already been forked. Ideally, @SteLeo1602 would have deleted their fork for this video so that viewers could see the entire process.

File: src/xdocs/beginning_development.xml [hunk 1]
```diff
@@ -49,6 +49,12 @@
         <a href="idea.html">IntelliJ IDEA IDE</a><br/>
       </p>
 
+      <p>
+        Video walk through of these instructions is available in this
+        <a href="https://www.youtube.com/playlist?list=PLHM9s_lN4X0hzOQ0sUmGdroxW0HfREAqj">
+           playlist</a>
```
Comment (rnveach) [inline src/xdocs/beginning_development.xml:55]: This should end with a period.

**Review by rnveach — CHANGES_REQUESTED** [review]

File: src/xdocs/beginning_development.xml [hunk 2]
```diff
@@ -49,6 +49,12 @@
         <a href="idea.html">IntelliJ IDEA IDE</a><br/>
       </p>
 
+      <p>
+        Video walk through of these instructions is available in this
```
Comment (rnveach) [inline src/xdocs/beginning_development.xml:53]: `through` => `throughs`

Comment (rnveach) [inline src/xdocs/beginning_development.xml:53]: `instructions is available` => `instructions are available`

**Review by rnveach — CHANGES_REQUESTED** [review]

File: src/xdocs/beginning_development.xml (hunk 1 above)
Comment (rnveach) [inline src/xdocs/beginning_development.xml:55]: `playlist` => `YouTube playlist`

"playlist on YouTube" is also acceptable.

**Review by rnveach — CHANGES_REQUESTED** [review]

File: src/xdocs/beginning_development.xml (hunk 2 above)
Comment (rnveach) [reply src/xdocs/beginning_development.xml:53]: `Video walk` => `Community video walk`

Comment (rnveach) [reply src/xdocs/beginning_development.xml:53]: `in this` => `in {your name here}`

Replace {your name here} with the name you go by on your Youtube channel.

**Comment by SteLeo1602** [comment]
> Thanks @SteLeo1602. My only possible item for improvement would to actually show how to fork the repo, but it is understandable to not do this with open PRs :)

**Comment by SteLeo1602** [comment]
> Thanks @SteLeo1602. My only possible item for improvement would to actually show how to fork the repo, but it is understandable to not do this with open PRs :)

I was going to show them the process of fork creation, but indeed decided to leave it in favor of existing pull requests. What I can try to do is to put the instructions for creating a fork over the moment of that existing video to display my thoughts.

File: src/xdocs/beginning_development.xml (hunk 1 above)
Comment (SteLeo1602) [reply src/xdocs/beginning_development.xml]: Done

File: src/xdocs/beginning_development.xml (hunk 2 above)
Comment (SteLeo1602) [reply src/xdocs/beginning_development.xml]: Done

Comment (SteLeo1602) [reply src/xdocs/beginning_development.xml]: Done

File: src/xdocs/beginning_development.xml (hunk 1 above)
Comment (SteLeo1602) [reply src/xdocs/beginning_development.xml]: Done

**Comment by SteLeo1602** [comment]
GitHub, generate website

**Review by rnveach — APPROVED** [review]

**Pull request merged**